### DISCUSSION TITLE
Dialog Placement and Auto-save Lane Changes

### DISCUSSION THREAD

**Discussion by dj69** [slyh/Cities2-TrafficLightsEnhancement#53] [Ideas]
Would it be possible to move the dialog box closer to center?  I play on a UWHD monitor and it's quite a jump back and forth when editing specific lanes, in order to save.  Also, would it be possible to "auto-save" lane edits so I can quickly click through each of them?  I'm often having to turn off lanes that allow all directions, esp.  when working with one-way streets or after having upgraded a road to more lanes.

**Comment by slyh** [comment]
The panel is now draggable in [v0.2.9-alpha](https://github.com/slyh/Cities2-TrafficLightsEnhancement/releases/tag/v0.2.9)!

Regarding "auto-save", I don't have a good idea in terms of UX. Unlike a webpage, you can't simply click outside a panel to close it, so a save button is still necessary.
